app/controllers/dashboard_controller.rb

```diff
@@ -74,7 +74,8 @@ def site_dash
     @site = Site.find(params[:site_id])
 
     @months = params[:months].to_s.empty? ? 3 : params[:months].to_i
-    @months_string = @months <= 1 ? 'month' : "#{@months} months"
+    @months < 1 && @months = 1
```

Comment:
wat-face.jpg

If this is meant to be a conditional assignment, make it an actual conditional assignment (`x = y if z == 1`, or with a full if statement)

Reply:
I think rubocop is known to complain about the full if statement and pointed me to the && one, but looks like it is fine with `... if ...` construct.